conda/plugins/virtual_packages/archspec.py

```diff
@@ -7,4 +7,6 @@
 def conda_virtual_packages():
     from ...core.index import get_archspec_name
 
-    yield CondaVirtualPackage("archspec", get_archspec_name())
+    archspec_name = get_archspec_name()
+    if archspec_name is not None:
+        yield CondaVirtualPackage("archspec", "1", archspec_name)
```

Comment:
Ooooh, that's an interesting one, not sure how this was lost, compared to https://github.com/conda/conda/blob/22.9.0/conda/core/index.py#L217-L219